src/com/pentalog/pentastagiu/homework2/shop/Basket.java

```diff
@@ -0,0 +1,55 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket {
+    private Customer customer;
+    private List<Product> product = new ArrayList<>();
+    private double totalPrice = 0;
+
+    public List<Product> getProduct() {
+        return product;
+    }
+
+    public void basketTotal(double price){
+        totalPrice += price;
+        System.out.println(totalPrice);
+    }
+
+    public String getPriceWithDiscount(Customer customer){
+        DecimalFormat df = new DecimalFormat(".##");
+        double percent = 0;
+        double total;
+        switch(customer.getMembership()){
+            case NOMEMBERSHIP:
+                percent = 0.0;
+                break;
+            case SILVER:
+                percent = 0.1;
```

Comment:
o imbunatatire pe care o poti face e sa muti percent direct in enum(field private in enum) pt ca e posibil sa tot apara logica in proiect legata de membership si ar trebui sa duplici pe peste tot percent = 0.1 iar daca la un moment dat vei vrea sa modifici aceasta valoare, o sa fie mai greu de modificat decat daca ai avea un loc centralizat in care sa fie exprimata aceasta relatie